ai_ta_backend/main.py

```diff
@@ -2,6 +2,7 @@
 import os
 import time
 from typing import List
+import json

```

Comment:
Let's remove this import, it appears to be unused.

Reply:
done!